### PR TITLE
Smoke test

### PR DESCRIPTION
**Pull request by philrz**
I expect to be updating this package frequently, since MaxMind puts out a new GeoLite2 database every Tuesday, and I'd like users to have the benefit of always having the latest as of when they install. @alfred-landrum had the good idea of creating some kind of smoke test that could easily tell me that everything is still functional after swapping a new DB. This would allow me to rationalize just merging my own DB-update PRs once the automation runs green.

This GitHub Actions smoke test does that by installing the package at the current SHA to an instance of Zeek and running a GeoLocation query against an IP that's believed to have a stable entry in the DB (it happens to be the one associated with my home Internet connection). Here's an example run of it failing when I told it to grep for the wrong location:

https://github.com/brimsec/geoip-conn/runs/896582430

Meanwhile, the run initiated by opening this PR should pass.

Per comments in the script, an unfortunate challenge I ran into is that [the binary distributions of Zeek](https://software.opensuse.org//download.html?project=security%3Azeek&package=zeek) are apparently not linked with libmaxminddb. That meant I couldn't just use one of those and install geoip-conn on top of it. Instead I'm compiling Zeek for now. So, that takes time, but at least it's not manual labor. I [asked on Zeek Slack](https://zeekorg.slack.com/archives/CSZBXF6TH/p1595372987384800?thread_ts=1594235715.230000&cid=CSZBXF6TH) if there'd be any interest in starting to link those binary packages to libmaxminddb, but no response yet. If my current approach here gets merged, maybe I'll open an official GitHub Issue against Zeek in hopes they'll do it and I'll be able to later circle back and make this faster.

### CODE REVIEW

File: smoketest.sh
Comment (alfred-landrum) [inline smoketest.sh]:
```suggestion
#!/bin/bash -ex
```

File: .github/workflows/smoketest.yml
Comment (alfred-landrum) [inline .github/workflows/smoketest.yml]: Is this `defaults` section needed? I don't see it on our other actions yamls.

**Review by alfred-landrum — APPROVED** [review]

File: .github/workflows/smoketest.yml
Comment (philrz) [reply .github/workflows/smoketest.yml]: Good eye. I'd used the [brim.yml](https://github.com/brimsec/zeek/blob/master/.github/workflows/brim.yml) from our Zeek fork as my template, and it happened to include that section. I'm assuming that had been there because the Zeek build action runs on both UNIX-style and Windows hosts, and the Windows ones have the option to run on alternates like PowerShell. You're right that it's surely needed, so I've just pushed a commit to remove it, and once everything runs clean I'll merge the PR.

**Comment by philrz** [comment]
Per my closing paragraph above, I've now opened https://github.com/zeek/zeek/issues/1086 in hopes the Zeek maintainers would be willing to start compiling the Zeek binary packages with libmaxminddb. If they move forward with that, I'll circle back and update this Action to use one of those binaries rather than burning time on compiling Zeek every time.

**Pull request merged**
